src/lib/e-net/index.ts

```diff
@@ -1,91 +1,56 @@
-import {ModelPluginInterface} from '../../interfaces';
-
-import {ERRORS} from '../../util/errors';
-import {buildErrorMessage} from '../../util/helpers';
+import {z} from 'zod';
 
+import {ModelPluginInterface} from '../../interfaces';
 import {ModelParams} from '../../types/common';
 
-const {InputValidationError} = ERRORS;
+import {validate} from '../../util/validations';
```

Comment:
validations can be imported before types

Reply:
done